frontend/src/constants/AuthConstants.ts

```diff
@@ -1,3 +1,10 @@
+import { getLocalStorageObjProperty } from "../utils/LocalStorageUtils";
+
 const AUTHENTICATED_USER_KEY = `${window.location.hostname}:AUTHENTICATED_USER`;
 
+export const BEARER_TOKEN = `Bearer ${getLocalStorageObjProperty(
```

Comment:
Was this part of another ticket? If so, just wanted to make sure this is not gonna create any conflicts on merge?

Reply:
We need this line for other tickets as well so it'll be fine to leave

Reply:
sounds good!